extras/src/test/java/com/google/gson/graph/GraphAdapterBuilderTest.java

```diff
@@ -129,8 +129,8 @@ public void testDeserializeListOfLists() {
   @Test
   public void testSerializationWithMultipleTypes() {
     Company google = new Company("Google");
-    new Employee("Jesse", google);
-    new Employee("Joel", google);
+    Employee unused1 = new Employee("Jesse", google);
+    Employee unused2 = new Employee("Joel", google);
```

Comment:
Might be good to add a comment here explaining this?
```suggestion
    // Employee constructor adds `this` to the given Company object
    Employee unused1 = new Employee("Jesse", google);
    Employee unused2 = new Employee("Joel", google);
```